_foundations/linux.md

```diff
@@ -12,65 +12,65 @@ sidebar:
   - image: "assets/images/linux.svg"
     image_alt: "Linux Foundation logo"
     text: >
-      Tax Status: 501C6  
+      Tax Status: The Linux Foundation uses multiple entity types, including 501C3, 501C4, and 501C6. The Linux Foundation is a 501C6.  
       <br/><br/>
-      Popular Projects: CNCF, Hyperledger, Node.js, Xen  
+      Popular Projects: Automotive Grade Linux, Cloud Foundry, Cloud Native Computing Foundation, Core Infrastructure Initiative, Hyperledger, Let's Encrypt, Node.js Foundation, ONAP, SPDX, Xen Project, Yocto Project  
       <br/><br/>
       Accepts New Projects: Yes  
 
 core:
-  name: "Linux Foundation"
+  name: "The Linux Foundation"
   newProjects: "yes"
   url: "https://www.linuxfoundation.org/"
-  legalName: "Linux Foundation"
+  legalName: "The Linux Foundation"
   location: "US"
   nonprofit: "501C6"
   softwareType: "various"
   licenses: "various"
   copyrightPolicy: ""
 governance:
-  boardSize: "21"
-  boardType: "sponsor, elected"
-  boardURL: "https://www.linuxfoundation.org/about/bylaws"
-  membership: "sponsor"
+  boardSize: "22"
+  boardType: "member, elected, community, at-large"
```

Comment:
Question: how are community members elected to the board?  Reading the bylaws, it seems that only Platinum, Gold, and Silver members (i.e. that require significant annual dues) can vote for the board.  Thus, in the schema I'm using, the membership and boardType are "sponsor", because you have to pay to play directly in the formal governance of the LF itself.
Or am I misreading the bylaws?

Reply:
The kernel community elects a rep.